terminus-ui/cohort-date-range/src/cohort-date-range.component.md

```diff
@@ -0,0 +1,95 @@
+<h1>Cohort Date Range</h1>
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Basic usage](#basic-usage)
+- [Event driven](#event-driven)
+- [Inputs](#inputs)
+  - [allowCustomDates](#allowCustomDates)
+  - [Disable the component](#disable-the-component)
+
+## Basic usage
+
+Pass in the cohorts:
+
+```html
+<ts-cohort-date-range
+  [cohorts]="cohorts"
+></ts-cohort-date-range>
+```
+
+```typescript
+  public cohorts = [{
+    display: 'Last 90 days',
+    range: {
+      start: startOfDay(subDays(new Date(), 90)),
+      end: currentDate,
+    },
+  }, {
+    display: 'Last full month',
+    range: {
+      start: startOfDay(subMonths(startOfMonth(currentDate), 1)),
+      end: endOfDay(subDays(startOfMonth(currentDate), 1)),
+    },
+  }, {
+    display: 'Custom dates',
+    range: {
+      start: '',
+      end: '',
+    },
+  }];
+```
+
+NOTE: The keys inside the passed in `cohorts` object has to be in the form of
```

Comment:
We should call out the interface they'll need so they don't create their own: `TsDateCohort`